installer/install.go

```diff
@@ -0,0 +1,397 @@
+	//ImageRepoPw       string `long:"repo-pw" required:"false" description:"Private Docker image repo URL" value-name:"PASSWORD"`
+	ClusterName       string `short:"c" long:"cluster-name" required:"true" description:"Private cluster name" value-name:"NAME"`
+	Token     		  string `short:"t" long:"token" required:"true" description:"Astra API token" value-name:"TOKEN"`
+	AcceptEula        bool   `long:"accept-eula" required:"true" description:"(flag) Accept End User License Agreement"`
+	AstraAccountId    string `short:"a" long:"account-id" required:"true" description:"Astra account ID" value-name:"ID"`
+	AstraUrl          string `short:"u" long:"astra-url" required:"false" default:"https://eap.astra.netapp.io" description:"Url to Astra. E.g. 'https://integration.astra.netapp.io'" value-name:"URL"`
+	SkipTlsValidation bool   `short:"z" long:"disable-tls" required:"false" description:"(flag) Disable TLS validation. TESTING ONLY."`
+	HostAliasIP string `long:"hostAliasIP" required:"false" description:"The IP of the Astra host. TESTING ONLY." value-name:"IP"`
+	HostAlias bool   `long:"hostAlias" required:"false" description:"(flag) Set to enable HostAliasIP. TESTING ONLY"`
+
+}
+
+const (
+	NatsImageName               = "nats"
+	ConnectorDefaultsConfigPath = "./astraconnector_defaults.yaml"
+	YamlOutputPath              = "./deployConfig.yaml"
+	OperatorYamlPath            = "./astraconnector_operator.yaml"
+	ConnectorNamespace 			= "astra-connector"
+	ConnectorOperatorNamespace  = "astra-connector-operator"
+)
+
+func main() {
+	formatter := &log.TextFormatter{
+		FullTimestamp: true,
+	}
+	log.SetFormatter(formatter)
+
+	var opts Options
+	var connectorConfig ConnectorConfig
+	_, err := flags.Parse(&opts)
+	checkFatalErr(err)
+
+	log.Info("Installing Astra Connector")
+
+	absPath, err := filepath.Abs(ConnectorDefaultsConfigPath)
+	checkFatalErr(err)
+	yamlFile, err := ioutil.ReadFile(absPath)
+	checkFatalErr(err)
+	err = yaml.Unmarshal(yamlFile, &connectorConfig)
+	checkFatalErr(err)
+
+	// input check
+	if opts.ImageTar != "" || opts.ImageRepo != "" {
+		if !(opts.ImageTar != "" && opts.ImageRepo != "") {
+			log.Fatal("error: 'image-tar' and 'image-repo' options must be used together")
+		}
+	}
+
+	// input check
+	if opts.HostAlias {
+		if opts.HostAliasIP == "" {
+			log.Fatal("error: 'hostAliasIP' must be defined if the 'hostAlias' flag is set ")
+		}
+	}
+
+	if opts.ImageTar != "" && opts.ImageRepo != "" {
+		dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+		checkFatalErr(err)
+
+		log.Info("Loading image tar")
+		loadedImages, err := loadImageTar(dockerClient, opts)
+		checkFatalErr(err)
+
+		log.Info("Tagging images")
+		taggedImages, err := tagImages(dockerClient, loadedImages, opts.ImageRepo)
+		checkFatalErr(err)
+
+		//todo: Fix the auth issue preventing this from succeeding
+		//log.Info("Pushing images")
+		//err = pushImages(dockerClient, &opts, taggedImages)
+		//checkFatalErr(err)
```

Comment:
If the docker push is not working, then the pods wont come up right? Is the push a manual operation currently?

Reply:
No, push is below. It's doing it via system call instead of using this package i have commented out